README.md

```diff
@@ -1,333 +1,65 @@
 # ContainerPilot
```

Comment:
I struggled with the hello world for a while,  but wasn't able to come up with one that was simple without being misleadingly simplistic. 

If you have a suggestion, though...